### DISCUSSION TITLE
Chains with all the same values

### DISCUSSION THREAD

**Discussion by hhegefeld** [lnccbrown/HSSM#556] [Q&A]
Hi, I'm new to using DDMs. I'm trying to model a dataset hierarchically with only the random effect of participant as a "null" model to test more complex models against. I used this command: `null_model = hssm.HSSM(data=ddm_data, hierarchical=True, prior_settings="safe", link_settings="log_logit")` on data that looks like:

<img width="336" alt="Screenshot 2024-08-16 at 1 50 59 PM" src="https://github.com/user-attachments/assets/02165ab0-1664-44d2-a7c6-7c810c7d68e7">

The sampling runs, albeit with errors outputted: "Sampling 4 chains for 1_000 tune and 3_000 draw iterations (4_000 + 12_000 draws total) took 2259 seconds.
There were 12000 divergences after tuning. Increase `target_accept` or reparameterize.
The rhat statistic is larger than 1.01 for some parameters. This indicates problems during sampling. See https://arxiv.org/abs/1903.08008 for details
The effective sample size per chain is smaller than 100 for some parameters.  A higher number is needed for reliable rhat and ess computation. See https://arxiv.org/abs/1903.08008 for details"

The problem occurs when I try to plot the traces: `az.plot_trace(null_model.traces)`. I get the error: "RuntimeWarning: invalid value encountered in divide
  hist_dens = hist / (hist.sum() * np.diff(bin_edges))" which seems to be caused by an error computing: `kernel_n = int(bw * 2 * np.pi)` ("OverflowError: cannot convert float infinity to integer"). 

After looking online and finding several relevant posts ([here](https://discourse.pymc.io/t/how-to-plot-trace-of-either-chol-corr-0-1-or-chol-corr-1-0-from-model-with-lkjcholeskycov-prior/11965/2) and [here](https://github.com/arviz-devs/ArviZ.jl/issues/98)), it seemed that the error may be caused "when all draws for a chain were identical". I looked at the posterior of the model, and in fact,  the subject standard deviation for z seems to do just that (for at least 3 of 4 chains).

<img width="706" alt="Screenshot 2024-08-16 at 1 57 07 PM" src="https://github.com/user-attachments/assets/4f681cbe-e061-480d-b9e7-f102aaab8df4">

Is this a fundamental error with my model or data, or is it likely caused by version incompatibilities, etc? Thank you in advance for any help!

Note: I'm running through a JupyterLab Notebook in a conda environment on a computing cluster, if that is helpful to know.

**Comment by AlexanderFengler** [comment]
Hi @hhegefeld ,

could you provide us with a bit more info on the data and output, to get a better idea of what's going on?

1. How many datapoints are there per subject?
2. Are there natural conditions that the model as is, ignores?
3. What are the values of the other parameters in the chains?


Some things to try naively (but better to work with more information):

1. One thing to try is to set `loglik_kind = 'approx_differentiable` 
2. Don't make the `t` parameter hierarchical. The `t` parameter is proving a bit troublesome, it's at least a good check to see if the problems can be directyl attributed to it. 
3. Try both the `centered` and `non-centered` parameterizations. (How many datapoints do you have per subject?)

**Reply by hhegefeld** [reply]
Yes, I'm happy to provide more info. Thanks for the preliminary suggestions - I'll look into them!

1. There are between 143-159 datapoints per subject.
2. There are no natural conditions per se. In this task, participants see a headshot of a peer and must predict whether that peer likes them or not (this is the decision process we want to capture), then they receive feedback about whether the peer actually likes them. We are interested in testing a model that looks at how feedback on the previous trial (Acceptance/Rejection) impacts decisions on the following trial. However, a [previous study](https://onlinelibrary.wiley.com/doi/abs/10.1111/jora.12999) with this task found that including prior feedback didn't improve over a null model, so I'm not sure we would expect it to matter much here.
3. Only the random effects SDs seem to have this problem (bolded below). Copied from the output:
**a_1|participant_id (chain, draw, participant_id__factor_dim) float32 -0.03085 0.01199 ... -0.01645**
array([[[-0.03084513,  0.01199205, -0.05472339, ..., -0.07997718,
         -0.01178461,  0.01095165],
        [-0.0308473 ,  0.01199202, -0.05472343, ..., -0.07997718,
         -0.01178453,  0.01095169],
        [-0.03084847,  0.01199214, -0.05472341, ..., -0.07997689,
         -0.01178447,  0.01095164],
        ...,
        [-0.03085029,  0.0119908 , -0.05472241, ..., -0.07997763,
         -0.01178557,  0.01095091],
        [-0.03085185,  0.01199072, -0.05472245, ..., -0.07997761,
         -0.01178564,  0.01095093],
        [-0.03085046,  0.01199075, -0.05472253, ..., -0.07997765,
         -0.01178567,  0.01095097]],

       [[ 0.01551195,  0.03614604, -0.04511661, ..., -0.07667991,
          0.05108837,  0.05126598],
        [ 0.01551195,  0.036146  , -0.04511663, ..., -0.07667991,
          0.05108838,  0.05126592],
        [ 0.01551195,  0.036146  , -0.04511663, ..., -0.07667991,
          0.05108838,  0.05126592], 
...
        [-0.0213673 ,  0.02276585,  0.04195554, ..., -0.09958882,
         -0.0052095 , -0.01938674],
        [-0.0213673 ,  0.02276585,  0.04195544, ..., -0.09958882,
         -0.00520948, -0.01938669],
        [-0.02136734,  0.02276583,  0.04195546, ..., -0.09958882,
         -0.00520949, -0.01938669]],

       [-0.0350493 ,  0.05711629,  0.00372274, ..., -0.1288201 ,
         -0.00978677, -0.0164463 ],
        [-0.0350493 ,  0.0571163 ,  0.00372274, ..., -0.1288201 ,
         -0.00978677, -0.0164463 ],
        [-0.03504929,  0.0571163 ,  0.00372274, ..., -0.1288201 ,
         -0.00978677, -0.0164463 ],
        ...,
        [-0.03504968,  0.05711611,  0.00372626, ..., -0.12882088,
         -0.00978598, -0.01644747],
        [-0.03504968,  0.05711611,  0.00372625, ..., -0.12882088,
         -0.00978598, -0.01644747],
        [-0.03504971,  0.05711611,  0.00372625, ..., -0.12882088,
         -0.00978598, -0.01644745]]], dtype=float32)
**a_1|participant_id_sigma (chain, draw) float32 0.3307 0.3307 ... 0.3201 0.3201**
**array([[0.33070347, 0.3307037 , 0.33070323, ..., 0.33070934, 0.33070955,
        0.33070955],
       [0.33166164, 0.33166164, 0.33166164, ..., 0.33166164, 0.33166164,
        0.33166164],
       [0.34759474, 0.34759474, 0.34759474, ..., 0.34759474, 0.34759474,
        0.34759474],
       [0.32009646, 0.32009646, 0.32009646, ..., 0.32009646, 0.32009646,
        0.32009646]], dtype=float32)**
**a_Intercept (chain, draw) float32 0.05007 0.05007 ... 0.053 0.053**
array([[0.05007283, 0.05007281, 0.05007295, ..., 0.05008044, 0.05008048,
        0.0500803 ],
       [0.05644157, 0.0564416 , 0.0564416 , ..., 0.05644222, 0.05644216,
        0.05644221],
       [0.06177736, 0.06177736, 0.06177737, ..., 0.06177993, 0.06177993,
        0.06178002],
       [0.05301604, 0.05301602, 0.05301603, ..., 0.05300498, 0.05300498,
        0.05300494]], dtype=float32)
**t_1|participant_id (chain, draw, participant_id__factor_dim) float32 -0.01734 0.01851 ... 0.008084**
array([[[-0.01734038,  0.01850535,  0.02498701, ...,  0.01032699,
          0.0042298 ,  0.0281819 ],
        [-0.01734037,  0.0185054 ,  0.02498702, ...,  0.01032699,
          0.00422983,  0.02818195],
        [-0.01734019,  0.0185054 ,  0.02498704, ...,  0.01032695,
          0.00422991,  0.02818195],
        ...,
        [-0.01733616,  0.01850411,  0.02498686, ...,  0.01032757,
          0.00423347,  0.02818185],
        [-0.0173362 ,  0.01850411,  0.02498689, ...,  0.01032757,
          0.00423348,  0.02818183],
        [-0.01733609,  0.01850403,  0.02498689, ...,  0.01032755,
          0.00423345,  0.02818188]],

       [ 0.05636656,  0.01693012,  0.00517394, ...,  0.04246807,
          0.00106519, -0.03804817],
        [ 0.05636656,  0.01693012,  0.00517394, ...,  0.04246804,
          0.00106519, -0.03804817],
        [ 0.05636656,  0.01693012,  0.00517394, ...,  0.04246804,
          0.00106519, -0.03804817],
...
        [ 0.03867628, -0.00738995,  0.03807021, ...,  0.08410902,
          0.02984077,  0.01337769],
        [ 0.03867637, -0.00738994,  0.0380703 , ...,  0.08410902,
          0.02984073,  0.01337771],
        [ 0.03867637, -0.00738993,  0.03807033, ...,  0.08410902,
          0.02984071,  0.01337771]],

       [ 0.03643293, -0.0138709 , -0.07465126, ...,  0.00589484,
          0.05655531,  0.00808306],
        [ 0.03643293, -0.0138709 , -0.07465126, ...,  0.00589483,
          0.05655532,  0.00808306],
        [ 0.03643293, -0.0138709 , -0.07465126, ...,  0.00589484,
          0.05655532,  0.00808306],
        ...,
        [ 0.0364331 , -0.01387242, -0.07465319, ...,  0.00589491,
          0.05655434,  0.00808435],
        [ 0.03643309, -0.01387242, -0.07465319, ...,  0.00589491,
          0.05655434,  0.00808435],
        [ 0.03643315, -0.01387238, -0.07465319, ...,  0.00589493,
          0.05655428,  0.00808435]]], dtype=float32)
**t_1|participant_id_sigma (chain, draw) float32 0.2501 0.2501 ... 0.3715 0.3715**
**array([[0.25010478, 0.25010478, 0.25010478, ..., 0.25010478, 0.25010478,
        0.25010478],
       [0.26479277, 0.26479277, 0.26479277, ..., 0.2647933 , 0.2647933 ,
        0.2647933 ],
       [0.29049966, 0.29049966, 0.29049966, ..., 0.29049966, 0.29049966,
        0.29049966],
       [0.3715048 , 0.3715048 , 0.3715048 , ..., 0.371505  , 0.371505  ,
        0.371505  ]], dtype=float32)**
**t_Intercept (chain, draw) float32 -0.6044 -0.6046 ... -0.6163 -0.616**
array([[-0.60438067, -0.60455585, -0.60448766, ..., -0.5758999 ,
        -0.5760838 , -0.5758693 ],
       [-0.6537143 , -0.6535486 , -0.6535486 , ..., -0.59118134,
        -0.590784  , -0.59081775],
       [-0.6583893 , -0.658675  , -0.65860116, ..., -0.6350715 ,
        -0.63574606, -0.63533586],
       [-0.6156779 , -0.61570174, -0.61567485, ..., -0.6162282 ,
        -0.6162698 , -0.61596394]], dtype=float32)
**v_1|participant_id (chain, draw, v_1|participant_id__factor_dim) float32 -0.01337 -0.05145 ... 0.002932**
array([[[-0.01337134, -0.05144778, -0.05970566, ...,  0.08751151,
          0.02538026, -0.01098117],
        [-0.01337132, -0.05144772, -0.05970566, ...,  0.08751155,
          0.02538033, -0.01098123],
        [-0.01337129, -0.05144766, -0.05970553, ...,  0.08751155,
          0.02538049, -0.01098134],
        ...,
        [-0.01336981, -0.05144763, -0.05970762, ...,  0.08751378,
          0.02537869, -0.0109805 ],
        [-0.0133698 , -0.05144763, -0.05970765, ...,  0.08751375,
          0.02537884, -0.01098049],
        [-0.01336987, -0.05144759, -0.05970769, ...,  0.08751367,
          0.02537874, -0.01098049]],

       [[-0.00280073,  0.0029414 ,  0.03181054, ...,  0.07654823,
         -0.01487516, -0.02973017],
        [-0.00280074,  0.00294141,  0.0318105 , ...,  0.07654826,
         -0.01487515, -0.02973017],
        [-0.00280074,  0.00294141,  0.0318105 , ...,  0.07654826,
         -0.01487515, -0.02973017],
...
        [-0.01170346, -0.15173058,  0.04129062, ...,  0.05324562,
         -0.0326575 , -0.06162395],
        [-0.01170346, -0.15173058,  0.04129067, ...,  0.05324562,
         -0.03265745, -0.06162406],
        [-0.01170345, -0.15173058,  0.04129067, ...,  0.05324566,
         -0.03265747, -0.0616241 ]],

       [[ 0.0333603 ,  0.00092741,  0.03221982, ...,  0.05720497,
         -0.05534797,  0.0029316 ],
        [ 0.0333603 ,  0.0009274 ,  0.03221982, ...,  0.05720497,
         -0.05534798,  0.00293161],
        [ 0.0333603 ,  0.0009274 ,  0.03221982, ...,  0.05720493,
         -0.05534799,  0.00293161],
        ...,
        [ 0.03336134,  0.00092826,  0.03222173, ...,  0.05721665,
         -0.05534912,  0.0029317 ],
        [ 0.03336134,  0.00092826,  0.03222173, ...,  0.05721668,
         -0.05534912,  0.0029317 ],
        [ 0.03336145,  0.00092825,  0.03222176, ...,  0.05721668,
         -0.05534912,  0.0029317 ]]], dtype=float32)
**v_1|participant_id_sigma (chain, draw) float32 0.3596 0.3596 ... 0.326 0.326**
**array([[0.35964495, 0.35964495, 0.35964495, ..., 0.3596492 , 0.3596492 ,
        0.3596492 ],
       [0.30349076, 0.30349076, 0.30349076, ..., 0.3034906 , 0.3034906 ,
        0.3034906 ],
       [0.35188583, 0.35188583, 0.35188583, ..., 0.35188583, 0.35188583,
        0.35188583],
       [0.3259803 , 0.3259803 , 0.3259803 , ..., 0.3259803 , 0.3259803 ,
        0.3259803 ]], dtype=float32)**
**v_Intercept (chain, draw) float32 0.09793 0.09793 ... 0.1217 0.1217**
array([[0.0979263 , 0.09792616, 0.09792652, ..., 0.09792557, 0.09792553,
        0.09792571],
       [0.13043122, 0.1304309 , 0.1304309 , ..., 0.13043912, 0.13043933,
        0.13043939],
       [0.13704255, 0.13704255, 0.13704255, ..., 0.13703918, 0.13703918,
        0.13703918],
       [0.1216582 , 0.12165818, 0.12165819, ..., 0.12165962, 0.12165963,
        0.12165963]], dtype=float32)
**z_1|participant_id (chain, draw, participant_id__factor_dim) float32 0.02399 -0.02304 ... -0.08653**
array([[[ 0.02399292, -0.02304072, -0.01436957, ..., -0.00940595,
         -0.00367659, -0.0141798 ],
        [ 0.02399292, -0.02304073, -0.01436991, ..., -0.00940594,
         -0.00367644, -0.01417983],
        [ 0.02399288, -0.02304073, -0.01437068, ..., -0.0094059 ,
         -0.00367716, -0.01417986],
        ...,
        [ 0.02399025, -0.02303812, -0.01436783, ..., -0.00940588,
         -0.00366938, -0.01418126],
        [ 0.02399026, -0.02303814, -0.01436773, ..., -0.00940584,
         -0.00366943, -0.01418127],
        [ 0.02399026, -0.02303815, -0.01436743, ..., -0.00940585,
         -0.00366951, -0.01418125]],

       [[ 0.00693948,  0.00310726,  0.01066911, ...,  0.01363245,
          0.01810646, -0.01268098],
        [ 0.00693948,  0.00310724,  0.01066909, ...,  0.01363244,
          0.01810646, -0.01268097],
        [ 0.00693948,  0.00310724,  0.01066909, ...,  0.01363244,
          0.01810646, -0.01268097],
...
        [ 0.01619649, -0.00792592, -0.03723641, ..., -0.01931195,
          0.01784155,  0.04017756],
        [ 0.01619643, -0.00792594, -0.03723641, ..., -0.01931195,
          0.01784151,  0.04017764],
        [ 0.01619642, -0.00792593, -0.03723641, ..., -0.01931195,
          0.01784151,  0.04017764]],

       [[-0.02886983, -0.0531828 ,  0.06303069, ...,  0.03752638,
         -0.01634057, -0.08653284],
        [-0.02886984, -0.05318279,  0.06303069, ...,  0.03752638,
         -0.01634057, -0.08653282],
        [-0.02886984, -0.0531828 ,  0.06303069, ...,  0.03752639,
         -0.01634057, -0.08653282],
        ...,
        [-0.02887061, -0.05318124,  0.06302979, ...,  0.03752533,
         -0.01633968, -0.08653148],
        [-0.02887062, -0.05318124,  0.06302979, ...,  0.03752533,
         -0.01633968, -0.08653147],
        [-0.02887062, -0.05318124,  0.06302979, ...,  0.03752533,
         -0.01633971, -0.08653136]]], dtype=float32)
**z_1|participant_id_sigma (chain, draw) float32 0.2732 0.2732 ... 0.3039 0.3039**
**array([[0.27317587, 0.27317587, 0.27317587, ..., 0.27317587, 0.27317587,
        0.27317587],
       [0.26986775, 0.26986775, 0.26986775, ..., 0.26986337, 0.26986337,
        0.26986337],
       [0.26611817, 0.26611817, 0.26611817, ..., 0.26611817, 0.26611817,
        0.26611817],
       [0.30386874, 0.30386874, 0.30386874, ..., 0.30386874, 0.30386874,
        0.30386874]], dtype=float32)**
**z_Intercept (chain, draw) float32 -0.03716 -0.03716 ... -0.08945**
array([[-0.03715702, -0.037157  , -0.03715713, ..., -0.03715317,
        -0.03715319, -0.03715319],
       [-0.05564386, -0.05564383, -0.05564383, ..., -0.0556422 ,
        -0.05564227, -0.05564226],
       [-0.07285464, -0.07285464, -0.07285464, ..., -0.07285525,
        -0.07285555, -0.07285555],
       [-0.08945432, -0.08945432, -0.08945432, ..., -0.08944868,
        -0.08944866, -0.08944858]], dtype=float32)
**v (chain, draw, rt,response_obs) float64 0.08455 0.08455 ... 0.1246 0.1246**
array([[[0.08455496, 0.08455496, 0.08455496, ..., 0.08694513,
         0.08694513, 0.08694513],
        [0.08455484, 0.08455484, 0.08455484, ..., 0.08694493,
         0.08694493, 0.08694493],
        [0.08455523, 0.08455523, 0.08455523, ..., 0.08694518,
         0.08694518, 0.08694518],
        ...,
        [0.08455575, 0.08455575, 0.08455575, ..., 0.08694506,
         0.08694506, 0.08694506],
        [0.08455573, 0.08455573, 0.08455573, ..., 0.08694504,
         0.08694504, 0.08694504],
        [0.08455584, 0.08455584, 0.08455584, ..., 0.08694522,
         0.08694522, 0.08694522]],

       [[0.12763049, 0.12763049, 0.12763049, ..., 0.10070105,
         0.10070105, 0.10070105],
        [0.12763017, 0.12763017, 0.12763017, ..., 0.10070074,
         0.10070074, 0.10070074],
        [0.12763017, 0.12763017, 0.12763017, ..., 0.10070074,
         0.10070074, 0.10070074],
...
        [0.12533572, 0.12533572, 0.12533572, ..., 0.07541523,
         0.07541523, 0.07541523],
        [0.12533572, 0.12533572, 0.12533572, ..., 0.07541513,
         0.07541513, 0.07541513],
        [0.12533573, 0.12533573, 0.12533573, ..., 0.07541508,
         0.07541508, 0.07541508]],

       [[0.1550185 , 0.1550185 , 0.1550185 , ..., 0.1245898 ,
         0.1245898 , 0.1245898 ],
        [0.15501849, 0.15501849, 0.15501849, ..., 0.12458979,
         0.12458979, 0.12458979],
        [0.15501849, 0.15501849, 0.15501849, ..., 0.1245898 ,
         0.1245898 , 0.1245898 ],
        ...,
        [0.15502096, 0.15502096, 0.15502096, ..., 0.12459132,
         0.12459132, 0.12459132],
        [0.15502097, 0.15502097, 0.15502097, ..., 0.12459133,
         0.12459133, 0.12459133],
        [0.15502108, 0.15502108, 0.15502108, ..., 0.12459133,
         0.12459133, 0.12459133]]])
**a (chain, draw, rt,response_obs) float64 1.019 1.019 1.019 ... 1.037 1.037**
array([[[1.01941374, 1.01941374, 1.01941374, ..., 1.06292494,
         1.06292494, 1.06292494],
        [1.01941151, 1.01941151, 1.01941151, ..., 1.06292495,
         1.06292495, 1.06292495],
        [1.01941046, 1.01941046, 1.01941046, ..., 1.06292505,
         1.06292505, 1.06292505],
        ...,
        [1.01941624, 1.01941624, 1.01941624, ..., 1.06293224,
         1.06293224, 1.06293224],
        [1.01941468, 1.01941468, 1.01941468, ..., 1.06293229,
         1.06293229, 1.06293229],
        [1.01941593, 1.01941593, 1.01941593, ..., 1.06293215,
         1.06293215, 1.06293215]],

       [[1.07460539, 1.07460539, 1.07460539, ..., 1.11372199,
         1.11372199, 1.11372199],
        [1.07460543, 1.07460543, 1.07460543, ..., 1.11372196,
         1.11372196, 1.11372196],
        [1.07460543, 1.07460543, 1.07460543, ..., 1.11372196,
         1.11372196, 1.11372196],
...
        [1.04124033, 1.04124033, 1.04124033, ..., 1.04330461,
         1.04330461, 1.04330461],
        [1.04124033, 1.04124033, 1.04124033, ..., 1.04330466,
         1.04330466, 1.04330466],
        [1.04124039, 1.04124039, 1.04124039, ..., 1.04330476,
         1.04330476, 1.04330476]],

       [[1.01812911, 1.01812911, 1.01812911, ..., 1.03724664,
         1.03724664, 1.03724664],
        [1.0181291 , 1.0181291 , 1.0181291 , ..., 1.03724662,
         1.03724662, 1.03724662],
        [1.01812911, 1.01812911, 1.01812911, ..., 1.03724662,
         1.03724662, 1.03724662],
        ...,
        [1.01811746, 1.01811746, 1.01811746, ..., 1.03723395,
         1.03723395, 1.03723395],
        [1.01811746, 1.01811746, 1.01811746, ..., 1.03723395,
         1.03723395, 1.03723395],
        [1.01811739, 1.01811739, 1.01811739, ..., 1.03723393,
         1.03723393, 1.03723393]]])
**z (chain, draw, rt,response_obs) float64 0.4967 0.4967 ... 0.4561 0.4561**
array([[[0.49670902, 0.49670902, 0.49670902, ..., 0.48716861,
         0.48716861, 0.48716861],
        [0.49670903, 0.49670903, 0.49670903, ..., 0.48716861,
         0.48716861, 0.48716861],
        [0.49670898, 0.49670898, 0.49670898, ..., 0.48716857,
         0.48716857, 0.48716857],
        ...,
        [0.49670932, 0.49670932, 0.49670932, ..., 0.48716921,
         0.48716921, 0.48716921],
        [0.49670931, 0.49670931, 0.49670931, ..., 0.4871692 ,
         0.4871692 , 0.4871692 ],
        [0.49670931, 0.49670931, 0.49670931, ..., 0.48716921,
         0.48716921, 0.48716921]],

       [[0.48782631, 0.48782631, 0.48782631, ..., 0.48292543,
         0.48292543, 0.48292543],
        [0.48782632, 0.48782632, 0.48782632, ..., 0.48292544,
         0.48292544, 0.48292544],
        [0.48782632, 0.48782632, 0.48782632, ..., 0.48292544,
         0.48292544, 0.48292544],
...
        [0.4858391 , 0.4858391 , 0.4858391 , ..., 0.4918313 ,
         0.4918313 , 0.4918313 ],
        [0.48583901, 0.48583901, 0.48583901, ..., 0.49183125,
         0.49183125, 0.49183125],
        [0.48583901, 0.48583901, 0.48583901, ..., 0.49183125,
         0.49183125, 0.49183125]],

       [[0.47045343, 0.47045343, 0.47045343, ..., 0.45611641,
         0.45611641, 0.45611641],
        [0.47045343, 0.47045343, 0.47045343, ..., 0.45611642,
         0.45611642, 0.45611642],
        [0.47045343, 0.47045343, 0.47045343, ..., 0.45611642,
         0.45611642, 0.45611642],
        ...,
        [0.47045464, 0.47045464, 0.47045464, ..., 0.45611815,
         0.45611815, 0.45611815],
        [0.47045464, 0.47045464, 0.47045464, ..., 0.45611816,
         0.45611816, 0.45611816],
        [0.47045466, 0.47045466, 0.47045466, ..., 0.4561182 ,
         0.4561182 , 0.4561182 ]]])
**t (chain, draw, rt,response_obs) float64 0.537 0.537 0.537 ... 0.5445 0.5445**
array([[[0.53701941, 0.53701941, 0.53701941, ..., 0.56203072,
         0.56203072, 0.56203072],
        [0.53692535, 0.53692535, 0.53692535, ..., 0.5619323 ,
         0.5619323 , 0.5619323 ],
        [0.53696206, 0.53696206, 0.53696206, ..., 0.56197062,
         0.56197062, 0.56197062],
        ...,
        [0.55253635, 0.55253635, 0.55253635, ..., 0.57826789,
         0.57826789, 0.57826789],
        [0.55243474, 0.55243474, 0.55243474, ..., 0.57816155,
         0.57816155, 0.57816155],
        [0.55255328, 0.55255328, 0.55255328, ..., 0.57828558,
         0.57828558, 0.57828558]],

       [[0.55026916, 0.55026916, 0.55026916, ..., 0.50069284,
         0.50069284, 0.50069284],
        [0.55036034, 0.55036034, 0.55036034, ..., 0.50077581,
         0.50077581, 0.50077581],
        [0.55036034, 0.55036034, 0.55036034, ..., 0.50077581,
         0.50077581, 0.50077581],
...
        [0.55079354, 0.55079354, 0.55079354, ..., 0.53703402,
         0.53703402, 0.53703402],
        [0.55042218, 0.55042218, 0.55042218, ..., 0.5366719 ,
         0.5366719 , 0.5366719 ],
        [0.55064801, 0.55064801, 0.55064801, ..., 0.53689209,
         0.53689209, 0.53689209]],

       [[0.56032127, 0.56032127, 0.56032127, ..., 0.54465929,
         0.54465929, 0.54465929],
        [0.56030791, 0.56030791, 0.56030791, ..., 0.54464631,
         0.54464631, 0.54464631],
        [0.56032297, 0.56032297, 0.56032297, ..., 0.54466095,
         0.54466095, 0.54466095],
        ...,
        [0.56001309, 0.56001309, 0.56001309, ..., 0.54436033,
         0.54436033, 0.54436033],
        [0.55998978, 0.55998978, 0.55998978, ..., 0.54433769,
         0.54433769, 0.54433769],
        [0.56016114, 0.56016114, 0.56016114, ..., 0.54450422,
         0.54450422, 0.54450422]]])

**Comment by AlexanderFengler** [comment]
@hhegefeld did you have any more luck on recent version of HSSM?
